shared/reducers/chat2.js

```diff
@@ -956,7 +956,7 @@ const rootReducer = (
       return state.set('staticConfig', action.payload.staticConfig)
     case Chat2Gen.metasReceived: {
       let nextState = action.payload.fromInboxRefresh ? state.set('inboxHasLoaded', true) : state
-      nextState = action.payload.initialTrustedLoad ? state.set('trustedInboxHasLoaded', true) : state
```

Comment:
maybe we should just move that into the withMutations (or just make it a merge) instead of this chaining